frontend/src/pages/projects/screens/detail/ProjectDetails.tsx

```diff
@@ -0,0 +1,48 @@
+import { Divider, PageSection, Stack, StackItem } from '@patternfly/react-core';
+import * as React from 'react';
+import { useParams } from 'react-router';
+import ApplicationsPage from '../../../ApplicationsPage';
+import EmptyProjects from '../../EmptyProjects';
+import DataConnectionsList from './DataConnectionsList';
+import ModelServingList from './ModelServingList';
+import ProjectDetailsSidebarWrapper from './ProjectDetailsSidebar';
```

Comment:
```suggestion
import ProjectDetailsSidebar from './ProjectDetailsSidebar';
```
No need to name it something other than what it is